packages/common/src/hooks/purchaseContent/usePurchaseContentFormConfiguration.ts

```diff
@@ -104,7 +104,7 @@ export const usePurchaseContentFormConfiguration = ({
     ? metadata?.track_id
     : undefined
 
-  const validationSchema = createPurchaseContentSchema(audiusQueryContext)
+  const validationSchema = createPurchaseContentSchema(audiusQueryContext, page)
```

Comment:
we may want to memoize this since its likely a somewhat expensive task of defining a schema every render?